packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -3594,6 +3594,20 @@ void main() {
     skip: kIsWeb, // [intended] Web uses its native context menu.
   );
 
+  testWidgets('Ensure consistent layout of SelectionArea children between platforms', (WidgetTester tester) async {
```

Comment:
This test should probably test that a SelectionArea wrapping a Text widget does not differ in alignment from a Text widget without a SelectionArea.

Something like below

```dart
Column(
    children: <Widget>[
        SelectionArea(child: Text()),
        Text(),
    ],
)
```

Both text widgets should begin at the same position on the x-axis.